src/qtism/data/storage/xml/marshalling/SimpleChoiceMarshaller.php

```diff
@@ -53,30 +57,41 @@ protected function unmarshallChildrenKnown(DOMElement $element, QtiComponentColl
                 $component->setFixed($fixed);
             }
 
-            if (Version::compare($version, '2.1.0', '>=') === true && ($templateIdentifier = $this->getDOMElementAttributeAs($element, 'templateIdentifier')) !== null) {
+            if (Version::compare(
+                    $version,
+                    '2.1.0',
+                    '>='
+                ) === true && ($templateIdentifier = $this->getDOMElementAttributeAs(
+                    $element,
+                    'templateIdentifier'
+                )) !== null) {
                 $component->setTemplateIdentifier($templateIdentifier);
             }
 
-            if (Version::compare($version, '2.1.0', '>=') === true && ($showHide = $this->getDOMElementAttributeAs($element, 'showHide')) !== null) {
+            if (Version::compare($version, '2.1.0', '>=') === true && ($showHide = $this->getDOMElementAttributeAs(
+                    $element,
+                    'showHide'
+                )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?